pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -239,7 +240,8 @@ func (configgen *ConfigGeneratorImpl) createGatewayHTTPFilterChainOpts(
 		// Build a filter chain for each TLS server
 		for i, server := range servers {
 			rdsName := model.GatewayRDSRouteName(server)
-			routeCfg := configgen.buildGatewayInboundHTTPRouteConfig(env, node, nameToServiceMap, gatewayNames, []*networking.Server{server}, rdsName)
+			routeCfg := configgen.buildGatewayInboundHTTPRouteConfig(env, node, push,
+				nameToServiceMap, gatewayNames, []*networking.Server{server}, rdsName)
```

Comment:
Please get rid of this fully. I had kept this here while adding RDS but forgot to remove these things after RDS started working.